src/frontend/components/registrationForm.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import Input from "./input";
+import Button from "./button";
+import Select from "./Select";
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  handleChange = () => {
+    console.log("Changes made in input ");
+  };
+  getDays = () => {
```

Comment:
This function is very ineffective in how its implemented. A simple for loop starting from 1 to 31 should be enough to generate all the days you need.